src/VisualStudio/Setup/Roslyn.VisualStudio.Setup.csproj

```diff
@@ -283,4 +251,11 @@
     <PackageReference Include="Microsoft.VisualStudio.Shell.Framework" Version="$(MicrosoftVisualStudioShellFrameworkVersion)" />
     <PackageReference Include="SQLitePCLRaw.bundle_green" Version="$(SQLitePCLRawbundle_greenVersion)" />
   </ItemGroup>
+  <ItemGroup>
+    <ServiceHubService Include="roslynCodeAnalysis" ClassName="Microsoft.CodeAnalysis.Remote.CodeAnalysisService" />
```

Comment:
@tmat so this is duplicated 2 times. should I create props file? or put in versions.props file?

...

and this name is in vsixmanifest file. is there a way to share name here and in vsixmanifest file?